server.py

```diff
@@ -137,6 +137,6 @@ def archives():
 if __name__ == "__main__":
     app.debug = True
 # Change the postgresql info below username, password, port
-    connect_to_db(app, "postgresql://username:password@localhost/newb")
-    app.run(port=5000)
+    connect_to_db(app, "postgresql:///newb")
+    app.run(port=5000, host='0.0.0.0')
```

Comment:
Should we keep these changes for everyone?

Reply:
Hi Ali! I apologize, "host='0.0.0.0'" was placed into the server.py since I'm using vagrant. That should be removed so that other users (not using vagrant) can run server.py on their local host (127.0.0.1). Same thing for "username:password@localhost". -Tien

Reply:
ah cool gotcha! feel free to push again to this PR to revert those changes, then we can merge!